my-project/src/Form/ImageTypeProduct.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\ImageProduct;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Validator\Constraints\File;
+
+class ImageTypeProduct extends AbstractType
+{
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('image',FileType::class,array(
+                'label' 	=> 'Dodaj obrazek',
```

Comment:
lets change all labels and texts to english to be consistent, remember to use text with traslation possibility